### ISSUE TITLE
[FEATURE] trouble detecting the "brightest" cells in image

### ISSUE THREAD

**Issue by EdwardNeuro** [MouseLand/cellpose#1271]
Hi, I'm using cellpose 3.1.1.2 to detect my cells but got some trouble. After training with my own 26 images and masks, it successfully improved precision, but couldn't detect the "brightest" cells, as is shown in attached screenshots, including one mannually and two auto detected.

I've changed my training parameters like learning rate and epochs, but got similar results. I'm wondering if anyone confronted similar problems and which steps I could make mistakes.

Thanks a million!

<img width="1753" height="1901" alt="Image" src="https://github.com/user-attachments/assets/b24582a6-dc1e-4c8b-8ea2-e3b06b28ea14" />
<img width="1777" height="1927" alt="Image" src="https://github.com/user-attachments/assets/331bb3ad-ca14-4ded-86bc-04aec67fdf06" />
<img width="1751" height="1897" alt="Image" src="https://github.com/user-attachments/assets/3ab09f66-7e6c-4e30-9112-77da5bebba41" />

**Comment by EdwardNeuro** [comment]
Thank you so much! I'll try it.

**Comment by EdwardNeuro** [comment]
Thank you so much, I've tried big data (more than 20GB) using cellpose.contrib.distributed on SAM-Cellpose, but it always caused out of GPU memory (NVIDIA RTX 5880 Ada 48G). That's why I turned to cp3😂and successfully run my code. It seems like cp3 can automatically adjust GPU usage, but SAMCP4 should be set somewhere manually? @mrariden

**Comment by mrariden** [comment]
If you have very large images the normalization may be thrown off which could be related to your issue. See #1277

Cellpose needs to make multiple copies of the image and holds multiple copies of intermediate representations all of which use memory. CP3 has a different architecture and a smaller model which is why you can use it on your data. I'd recommend experimenting with smaller images/crops/smaller data types.

**Comment by carsen-stringer** [comment]
We have now introduced a vitb model in Cellpose4, `cpdino-vitb` and this will use less GPU memory. `pip install git+https://github.com/mouseland/cellpose` and `pip install git+https://github.com/facebookresearch/dinov3`

Let us know if you still have questions
